apps/foundation-demo/src/app/app-routing.module.ts

```diff
@@ -27,6 +29,13 @@ export const appRoutes: Routes = [
         path: 'checkout', 
         component: CheckoutViewComponent 
       },
+      { 
+        canActivate: [
+          ThankYouGuard
+        ],
+        path: 'thank-you/:id', 
```

Comment:
We shouldn't redirect to an `id`. Additionally, I'd expect that the route was `/checkout/thank-you`.

Reply:
How exactly would this work? If you were trying to go directly to `/checkout/thank-you`, would it redirect to `/cart`? Wouldn't that mean you could navigate directly to `/checkout/thank-you` when there was an item in the cart?

Reply:
I don't think so. I believe the whole checkout route group should be guarded in the same way, but additional, thank you has the additional thank-you guard.

Reply:
Added an issue to address the changes we spoke about: https://github.com/graycoreio/daffodil/issues/203

Right now, the route is `/checkout/thank-you`, but there is no additional route guarding yet.